postgres-to-snowflake-with-cdc/postgres/replication-config.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE omd_customers REPLICA IDENTITY FULL;
+ALTER TABLE omd_pets REPLICA IDENTITY FULL;
+ALTER TABLE omd_appointments REPLICA IDENTITY FULL;
+
+SELECT oid::regclass, relreplident FROM pg_class
+ WHERE oid in ( 'omd_customers'::regclass, 'omd_pets'::regclass, 'omd_appointments'::regclass);
```

Comment:
What's the purpose of having this select here?

Reply:
just to verify that it's worked. do you think I should remove it?

Reply:
Just a bit odd to have a select in a set-up script and not doing anything with the result. I'd either remove it, or put a comment clarifying its purpose. Or have sort of an assert which fails if it doesn't return the expected state (just kidding, don't do that ;)

Reply:
Done 👍